app/assets/stylesheets/views/mod-center.scss

```diff
@@ -0,0 +1,143 @@
+@import '../config/import';
+
+.mod-center-heading {
+  margin-top: var(--su-6);
+  margin-bottom: var(--su-6);
+}
+
+.mod-index-header {
+  @media (min-width: $breakpoint-m) {
+    padding: var(--su-2) 0 var(--su-6);
+  }
+}
+
+.mod-index-list-header,
+.moderation-single-article {
+  border: none;
+  box-shadow: 0 0 0 1px var(--card-border);
+}
+
+.mod-index-list-header {
+  background-color: var(--base-0);
+  display: grid;
+  grid-column-gap: var(--su-2);
+  padding: var(--su-2) var(--su-4);
+
+  @media (min-width: $breakpoint-m) {
+    grid-template-columns: 4fr 2fr 1fr;
+  }
+
+  &__label {
+    font-size: var(--fs-s);
+    font-weight: var(--fw-medium);
+    color: var(--base-50);
+  }
+}
+
+.mod-index-list {
+  margin: 0;
+  width: 100%;
+}
+
+.moderation-single-article {
+  align-items: center;
+  background-color: var(--base-inverted);
+  display: grid;
+  grid-column-gap: var(--su-2);
+  grid-template-columns: 1fr;
+  grid-template-rows: auto;
+  grid-row-gap: var(--su-1);
+  padding: var(--su-4);
+  text-align: left;
+  width: 100%;
+
+  @media (min-width: $breakpoint-m) {
+    grid-template-columns: 4fr 2fr 1fr;
+    grid-row-gap: 0;
+    grid-template-rows: 1fr;
+  }
+
+  &:hover {
+    background: var(--base-0);
+
+    .article-title {
+      color: var(--link-brand-color);
+    }
+  }
+
+  .article-title--label {
+    font-weight: var(--fw-bold);
+    font-size: var(--fs-base);
+    line-height: var(--lh-tight);
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
```

Comment:
This is probably not related to your PR but do you mind updating this? I noticed in both mobile and desktop views `nowrap` for long titles doesn't work that well:

Mobile view:
![Screen Shot 2020-10-09 at 2 31 20 PM](https://user-images.githubusercontent.com/17884966/95619280-5aa4d700-0a3c-11eb-9bc6-95531f39fc20.png)

Desktop view:
![Screen Shot 2020-10-09 at 2 33 48 PM](https://user-images.githubusercontent.com/17884966/95619333-78723c00-0a3c-11eb-8598-853febac28f4.png)

Reply:
Resolving it in my next commit.